src/Network/Receive.pm

```diff
@@ -2896,24 +2900,41 @@ sub chat_join_result {
 
 sub chat_modified {
 	my ($self, $args) = @_;
+	
+	my $msg = $args->{RAW_MSG};
+	my $msg_size = $args->{RAW_MSG_SIZE};
+	my $header_pack = 'v a4 a4 v2 C';
+	my $header_len = ((length pack $header_pack) + 2);
+	
+	my $title_len = ($args->{len} - $header_len);
+	
+	my $title = substr($msg, $header_len, $title_len);
```

Comment:
`$title_len` is unnecessary. `substr $str, $offset` returns the rest of the string, so this is just:

```
my $title = bytesToString(substr $msg, $header_len);
```

I believe `bytesToString` is necessary for utf8 support.